src/Card/CardSection/CardSectionTitle.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { HEADING_SIZES_MAP } from '../constants';
+
+const CardSectionTitle = ({
+  as: Component,
+  className,
+  children,
+  ...attrs
+}) => (
+  <Component
+    {...attrs}
+    className={classNames('pgn__card-section-title', className)}
+  >
+    {children}
+  </Component>
+);
+
+CardSectionTitle.propTypes = {
+  as: PropTypes.elementType,
+  className: PropTypes.string,
+  children: PropTypes.node.isRequired,
+};
+
+CardSectionTitle.defaultProps = {
+  as: HEADING_SIZES_MAP.H5,
```

Comment:
Generally a fan of using constants; may be overkill here though. I created the `HEADING_SIZES_MAP` variable because I didn't want to hardcode heading values into the component itself for some conditionals.

Reply:
In this case the constants are redundant, you couldn't change the value of the constant without changing its name and they're unlikely to need to change since that part of the html spec is settled. In this case I wouldn't consider "h5" a [magic number](https://moderatemisbehaviour.github.io/clean-code-smells-and-heuristics/#g25-replace-magic-numbers-with-named-constants).